web-tester/pom.xml

```diff
@@ -0,0 +1,126 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.axonivy.ivy.webtest</groupId>
+  <artifactId>web-tester</artifactId>
+  <version>8.0.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  
+  <name>web-tester</name>
+  <description>API that simplifies web testing with ivy projects.</description>
+  <url>https://github.com/ivy-supplements/web-tester/</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>IvyTeam</name>
+      <email>support@ivyteam.ch</email>
+      <organization>AXON Ivy AG</organization>
+      <organizationUrl>https://www.axonivy.com</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:ivy-supplements/web-tester.git</connection>
+    <developerConnection>scm:git:git@github.com:ivy-supplements/web-tester.git</developerConnection>
+    <url>git@github.com:ivy-supplements/web-tester.git</url>
+  </scm>
+
+  <organization>
+    <name>The open source community of Axon.ivy</name>
+    <url>https://github.com/ivy-supplements</url>
+  </organization>
+  
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <name>Maven Central Snapshots Repo</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.axonivy.ivy.webtest</groupId>
+      <artifactId>primeui-tester</artifactId>
+      <version>7.0.0-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.1</version>
+        <configuration>
+          <source>11</source>
```

Comment:
ẁork with properties as soon as you have to duplicate vars .... 
however, I think that the new <release>11</release> config param would also be sufficient.

Reply:
you mean <release>11</release>?

Reply:
yes, go for <release>11</release> or introduce something like 
<source>${java.version}</source>
<target>${java.version}</target>

Reply:
now I see that github swallowed `<xml> t`ags in my initial comment. gotta use explicit code formattings in the future :)

Reply:
Had same problems in the comment above :D